build/tasks/aria-supported.js

```diff
@@ -46,18 +46,30 @@ module.exports = function(grunt) {
 					.reduce((out, [key] = item) => {
 						switch (listType) {
 							case 'supported':
-								if (subject.includes(key)) {
+								if (
+									subject.hasOwnProperty(key) &&
+									subject[key].unsupported === false
+								) {
 									out.push([`${key}`, 'Yes']);
 								}
 								break;
 							case 'unsupported':
-								if (!subject.includes(key)) {
+								if (
+									(subject[key] && subject[key].unsupported === true) ||
+									!subject.hasOwnProperty(key)
+								) {
 									out.push([`${key}`, 'No']);
 								}
 								break;
 							case 'all':
 							default:
-								out.push([`${key}`, subject.includes(key) ? 'Yes' : 'No']);
+								out.push([
+									`${key}`,
```

Comment:
What's this template string for? This could just be `out.push(key, ...)`.

Reply:
@stephenmathieson @dylanb 
`out` is what eventually becomes the array to pass to mdTable like so - https://github.com/dequelabs/axe-core/blob/develop/build/tasks/aria-supported.js#L78

And the generated output is here - https://github.com/dequelabs/axe-core/blob/develop/doc/aria-supported.md

Reply:
Right, but why are we wrapping a variable in backticks here? Consider the following:

```js
var a = `${out}`
var b = out
```